Musicplayer.css

```diff
@@ -67,62 +41,152 @@ li{
 }
 
 
+
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
-	color:black;
+	font-family: 'Open Sans', sans-serif;
```

Comment:
21/9: I change the fonts for my link in the nav bar to Open San which was the fonts that I decided to use for normal text. Also I make the text to be bold so It can be clearly read by all user(for both old and young people). Lastly because I make the background color for the nav bar to navy blue so I change the color of those text to white which make it display a great contrast for user.